### PR TITLE
Informational overlay revamps

### PR DESCRIPTION
**Pull request by mousewax**
Revamps to both the About page and the Connection Lost page to just add a little polish.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3980/pr-test (or any other room on that server)

**Comment by bjalder26** [comment]
Maybe this table could use some padding.
image

**Comment by mousewax** [comment]
That table took me by surprise. It's not on my local copy.

**Comment by bjalder26** [comment]
> That table took me by surprise. It's not on my local copy. And I have no idea why it's on this screen.

It's part of the new beta servers update.

**Comment by mousewax** [comment]
image

How is this?

**Comment by bjalder26** [comment]
In my opinion it all looks really nice.

**Comment by 96LawDawg** [comment]
I agree.  Looks really nice.  What is the source of the plug graphic?  Is attribution required?

**Comment by mousewax** [comment]
Attribution is not required. It's new work made by me from stock images that I own.

File: client/css/overlays/about.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+
+#aboutOverlay .header .about_joinus{
+  text-align: center;
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  width: 100%;
+  margin:;
```
Comment (RaphaelAlvez) [inline client/css/overlays/about.css:47]: : ;

**Review by RaphaelAlvez — CHANGES_REQUESTED** [review]

**Comment by 96LawDawg** [comment]
> Attribution is not required. It's new work made by me from stock images that I own.

@ArnoldSmith86, how do you want to handle that?  Most of the attribution notes are in the variants.  But this wouldn't fit there.  Put it on the attribution page with the comment about the deck?

**Comment by RaphaelAlvez** [comment]
> @ArnoldSmith86, how do you want to handle that? Most of the attribution notes are in the variants. But this wouldn't fit there. Put it on the attribution page with the comment about the deck?

I think this is pretty straightforward. @mousewax should add an attribution paragraph for all the assets they created if that's what they want.
My assets were never on that page because I didn't feel the need for it

**Comment by 96LawDawg** [comment]
It isn't about protecting mousewax.  It is about protecting VTT.  Listing the source of all graphics or other potentially copyrighted material helps prevent issues down the road.

**Comment by robartsd** [comment]
> It isn't about protecting mousewax. It is about protecting VTT. Listing the source of all graphics or other potentially copyrighted material helps prevent issues down the road.

If his license for the stock images allow him to create from them without restrictions, can't he put his work into the public domain?

**Comment by 96LawDawg** [comment]
Yes.  And he says he wants to maximize his stuff in the public domain.  So we just need to say it is a VTT asset and is in the public domain.  But my point is we need to say that somewhere so nobody looks at that and asks where did that cool cable come from?

**Comment by robartsd** [comment]
On the about overlay we have:

### Copyright Attribution
The default card deck was created by Adrian Kennard and released under CC0 Public Domain license: https://www.me.uk/cards/.

Perhaps it should be:
### Copyright Attribution
All images served under https://virtualtabletop.io/i/ are available from the GitHub repository under an open source license.

**Comment by ArnoldSmith86** [comment]
The connection lost overlay flashes for me quickly when I (re)load the page. Is it just me or does this only happen with this PR?

**Comment by robartsd** [comment]
> The connection lost overlay flashes for me quickly when I (re)load the page. Is it just me or does this only happen with this PR?

I think I've seen that in production occasionally as well.

**Comment by mousewax** [comment]
Would having a paragraph somewhere saying: 

All graphics used on this site were either created by members of the VTT development team or sourced from works in the public domain. Great care has been made to attribute the sources where appropriate. If you feel your work has been appropriated or used without your consent, please wrote to LawDawg because he's the one that said we could use it.

(The last sentence is of course humor, but something along these lines.)

**Comment by RaphaelAlvez** [comment]
> Would having a paragraph somewhere saying: 
> 
> All graphics used on this site were either created by members of the VTT development team or sourced from works in the public domain. Great care has been made to attribute the sources where appropriate. If you feel your work has been taken or used without your consent, please write to LawDawg because he's the one that said we could use it.
> 
> (The last sentence is of course humor, but something along these lines.)

No no. That is totally correct. We should keep it

**Comment by 96LawDawg** [comment]
LawDawg left the chat.  JohnSmith96 joined the chat.

**Comment by robartsd** [comment]
> All graphics used on this site were either created by members of the VTT development team or sourced from works in the public domain.

This is not a true statement. It is not even true for https://virtualtabletop.io/i/ (some assets there are CC BY). Public library contains assets that are CC BY NC. Anyone can upload files and have them served under https://virtualtabletio.io/assets/

**Comment by mousewax** [comment]
> All graphics used on this site were either created by members of the VTT development team or sourced from works in the public domain or made available for public use through Creative Commons or other similar licenses. Great care has been made to attribute the sources where appropriate. If you feel your work has been taken or used without your consent, please contacts....

**Comment by RaphaelAlvez** [comment]
> > All graphics used on this site were either created by members of the VTT development team or sourced from works in the public domain or made available for public use through Creative Commons or other similar licenses. Great care has been made to attribute the sources where appropriate. If you feel your work has been taken or used without your consent, please contact us and immediate action will be taken to remove any infringing works.

I like this. @96LawDawg and @robartsd how do you feel about it?

**Comment by robartsd** [comment]
> > > All graphics used on this site were either created by members of the VTT development team or sourced from works in the public domain or made available for public use through Creative Commons or other similar licenses. Great care has been made to attribute the sources where appropriate. If you feel your work has been taken or used without your consent, please contact us and immediate action will be taken to remove any infringing works.
> 
> I like this. @96LawDawg and @robartsd how do you feel about it?

Would like some language indicating where they can find the assets with associated licenses so that we meet any "share alike" clauses of licenses of the assets that we use.

**Comment by 96LawDawg** [comment]
We need a way to distinguish things "in the code" and things "in the library."  For stuff in the library, the current copyright attribution language works I think.

For stuff in the code, something like: "All graphics used on this site were either created by members of the VTT community and are released into the public domain or sourced from works in the public domain or made available for public use through Creative Commons or other similar licenses."  Stuff in the code includes the cable that started this discussion and VTT-Symbols.  I'm not aware of anything else.

Then an overall statement that says "Great care has been made to attribute the sources where appropriate. If you feel your work has been taken or used without your consent, please contact us and immediate action will be taken to remove any infringing works."

I'm not worried about random things that show up in /assets/ with checksum names and that are only linked to games that aren't in the public library.

**Comment by mousewax** [comment]
Tell me what words to put in which places and I will make it happen.

**Review by ArnoldSmith86 — APPROVED** [review]
I don't see problems in the changed code. I'm not 100% sure if the template tags work in every browser we support.

I really like the polish on the about overlay.

**Review by 96LawDawg — APPROVED** [review]
Approve with the comments noted above.

**Pull request merged**